### PR TITLE
add wickedone/phrase-tag-bundle

### PR DESCRIPTION
**Pull request by wickedOne**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | [wickedone/phrase-tag-bundle](https://packagist.org/packages/wickedone/phrase-tag-bundle)

### CODE REVIEW

**Comment by fabpot** [comment]
Can you remove unrelated changes?

File: wickedone/phrase-tag-bundle/2.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "WickedOne\\PhraseTagBundle\\WickedOnePhraseTagBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    }
```
Comment (diimpp) [inline wickedone/phrase-tag-bundle/2.0/manifest.json:7]: Just a suggestion, you can offer to configure DSN as well.
image

```suggestion
    "copy-from-recipe": {
        "config/": "%CONFIG_DIR%/"
    },
    "env": {
        "PHRASE_DSN": "phrase://PROJECT_ID:API_TOKEN@default?userAgent=myProject"
    }
```

**Review by diimpp — CHANGES_REQUESTED** [review]
Please remove unrelated changes as Fabian mentioned.

**Comment by wickedOne** [comment]
- removed unrelated changes (sorry, no idea how they managed to get in there)
- implemented `PHRASE_DSN` env var as suggested by @diimpp

File: wickedone/phrase-tag-bundle/2.0/config/packages/wickedone.yaml
Comment (diimpp) [inline wickedone/phrase-tag-bundle/2.0/config/packages/wickedone.yaml:1]: Missed it at first, file name should follow bundle name,

```patch
--- wickedone.yaml
+++ wicked_one_phrase_tag.yaml
```

Comment (wickedOne) [reply wickedone/phrase-tag-bundle/2.0/config/packages/wickedone.yaml:1]: done

**Review by diimpp — APPROVED** [review]

**Pull request merged**